cyw43/src/consts.rs

```diff
@@ -51,6 +65,13 @@ pub(crate) const REG_BACKPLANE_READ_FRAME_BC_HIGH: u32 = 0x1001C;
 pub(crate) const REG_BACKPLANE_WAKEUP_CTRL: u32 = 0x1001E;
 pub(crate) const REG_BACKPLANE_SLEEP_CSR: u32 = 0x1001F;
 
+pub(crate) const I_HMB_SW_MASK: u32 = (0x000000f0);
```

Comment:
let me know if these shouldn't just be sitting here/should be clenaed up to a different group or something

Reply:
no strong preference, up to you